From 7b68e6271c0d65aaf13b11f96df43d8572db22fb Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Wed, 3 Jul 2019 17:46:53 +0000
Subject: [PATCH] OS-7879 zfs recv is broken Reviewed by: Kody Kantor
 <kody.kantor@joyent.com> Reviewed by: Jason King <jason.king@joyent.com>
 Approved by: Kody Kantor <kody.kantor@joyent.com>

---
 usr/src/lib/libzfs/common/libzfs_sendrecv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/usr/src/lib/libzfs/common/libzfs_sendrecv.c b/usr/src/lib/libzfs/common/libzfs_sendrecv.c
index 27f05f101e..2bf7c160de 100644
--- a/usr/src/lib/libzfs/common/libzfs_sendrecv.c
+++ b/usr/src/lib/libzfs/common/libzfs_sendrecv.c
@@ -3754,7 +3754,6 @@ zfs_receive_one(libzfs_handle_t *hdl, int infd, const char *tosnap,
 		}
 
 		newfs = B_TRUE;
-		*cp = '/';
 	}
 
 	zc.zc_begin_record = *drr_noswap;
-- 
2.21.0

